src/proto/weights.proto

```diff
@@ -51,6 +52,8 @@ message Initializer {
   }
 
   // Weight initializers
+  message LayerDefault {}
```

Comment:
If no initializer is set, we don't use a "layer default". The weights object might not be associated with any layer, or with multiple layers. The actual default is zero initialization.

I can buy the argument that users should specify every `oneof` field within a message, but I don't think it's the right approach for users to specify every field that contains a `oneof` field. For instance:
```
weights {
  name: "my_weights"
}
```
should instantiate weights with zero initialization and the model's default optimizer (in the future it might be better to instantiate no optimizer).

Reply:
Got it. Will remove this. Since the default already has a name, no sense naming it twice.

Reply:
I thought that we used a default, e.g. for convolution, a He initializer. Or is that only if you don't specify the weights at all?

Reply:
If a layer gets to create a weights, it can choose a suitable initializer. If a weights is created in the void, then we can't make any assumptions and we should use the most inoffensive initialization scheme.

Reply:
`LayerDefault` has been removed. Thanks for the clarifications.